scrapers/Reddit.yml

```diff
@@ -0,0 +1,57 @@
+name: "Reddit"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - reddit.com
+      - old.reddit.com
```

Comment:
This will never be matched: if an url contains `old.reddit.com` the above pattern will have already succeeded because it also contains `reddit.com` 🙂
```suggestion
```

Reply:
Obviously. Genuine accident leaving this in as I was experimenting with reddit's individual subdomains before seeing it would work across all of them and forgot to remove that 😬